django/db/backends/oracle/schema.py

```diff
@@ -42,6 +45,42 @@ def quote_value(self, value):
         else:
             return str(value)
 
+    def create_model(self, model):
+        super().create_model(model)
+        if model._meta.db_table_comment:
+            self.execute(
+                self.sql_alter_table_comment
+                % {
+                    "table": model._meta.db_table,
+                    "comment": self.quote_comment_value(model._meta.db_table_comment),
+                },
+                [],
+            )
+        for field in model._meta.local_fields:
+            if field.db_comment:
+                self.execute(
+                    self.sql_alter_column_comment
+                    % {
+                        "table": model._meta.db_table,
+                        "column": field.db_column or field.get_attname(),
+                        "comment": self.quote_comment_value(field.db_comment),
+                    },
+                    [],
+                )
+
+    def add_field(self, model, field):
+        super().add_field(model, field)
+        if field.db_comment and not field.many_to_many:
+            self.execute(
+                self.sql_alter_column_comment
+                % {
+                    "table": model._meta.db_table,
+                    "column": field.db_column or field.get_attname(),
```

Comment:
Why not `field.column`?
```suggestion
                    "column": field.column,
```